### ISSUE TITLE
Unsupported cuda toolkit version: `12.6`

### ISSUE THREAD

**Issue by MJE10** [coreylowman/cudarc#280]
I am new to cudarc, and trying to run an example project from candle (https://huggingface.github.io/candle/guide/installation.html). I just installed CUDA today, and it seems that 12.6 might be very new? The error message says to open a github issue, and I don't see anything currently open for this; please advise if this is not the right place to report it.

```
PS C:\> nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2024 NVIDIA Corporation
Built on Fri_Jun_14_16:44:19_Pacific_Daylight_Time_2024
Cuda compilation tools, release 12.6, V12.6.20
Build cuda_12.6.r12.6/compiler.34431801_0
PS C:\> nvidia-smi --query-gpu=compute_cap --format=csv
compute_cap
7.5
```

cargo build output:

```
error: failed to run custom build command for `cudarc v0.11.6`

Caused by:
  process didn't exit successfully: `C:\...` (exit code: 101)
  --- stdout
  cargo:rerun-if-changed=build.rs
  cargo:rerun-if-env-changed=CUDA_ROOT
  cargo:rerun-if-env-changed=CUDA_PATH
  cargo:rerun-if-env-changed=CUDA_TOOLKIT_ROOT_DIR

  --- stderr
  thread 'main' panicked at C:\...\cudarc-0.11.6\build.rs:79:14:
  Unsupported cuda toolkit version: `12.6`. Please raise a github issue.
```

**Comment by MJE10** [comment]
Confirming that the issue is the same for 0.11.8:

Cargo.toml:

```
[package]
name = "myapp"
version = "0.1.0"
edition = "2021"

[dependencies]
cudarc = { version = "=0.11.8", features = ["std", "cublas", "cublaslt", "curand", "driver", "nvrtc", "f16", "cuda-version-from-build-system", "dynamic-linking"], default-features=false }
```

Cargo build output:

```
error: failed to run custom build command for `cudarc v0.11.8`

Caused by:
  process didn't exit successfully: `C:\...\build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-changed=build.rs
  cargo:rerun-if-env-changed=CUDA_ROOT
  cargo:rerun-if-env-changed=CUDA_PATH
  cargo:rerun-if-env-changed=CUDA_TOOLKIT_ROOT_DIR

  --- stderr
  thread 'main' panicked at C:\...\cudarc-0.11.8\build.rs:82:14:
  Unsupported cuda toolkit version: `12.6`. Please raise a github issue.
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by maulberto3** [comment]
I am having the same issue. To ensure my cuda 12.5 works well, I did a small pure cuda code script and it does indeed. When pointing this same cuda toolkit to candle to run models with cuda, it errors, same as above.

**Comment by coreylowman** [comment]
Released on crates.io with new version of cudarc

**Comment by David-OConnor** [comment]
Does anyone know how to fix this, when upgrading from cudarc 0.10.0 to 0.12.1? Is there a code change that needs to be made?
